packages/webapp/src/routes/developer/clients-name.js

```diff
@@ -36,6 +40,12 @@ const handlers = {
 
     request.yar.set(constants.redisKeys.DEVELOPER_CLIENTS_NAME, { type: 'individual', value: { firstName, middleNames, lastName } })
 
+    processDeveloperTask(request,
```

Comment:
I've put this here so a user can complete the task list to get to the check and submit page. But it will be updated when the tasklist itself is updated.

Reply:
please add TODO as discussed

Reply:
TODO added